rakelib/artifacts.rake

```diff
@@ -43,7 +43,17 @@ namespace "artifact" do
     end.flatten.uniq
   end
 
-  task "prepare" => ["bootstrap", "plugin:install-default"]
+  task "use-defaults-gemfile" do
+    FileUtils.cp("Gemfile.defaults", "Gemfile")
```

Comment:
The copy is only done when calling a task of the `artifact:*` family.
The freezing is currently a human process before an official release.